### PR TITLE
Change logo and design

### PR DESCRIPTION
**Pull request by naotospace**

### CODE REVIEW

File: app/views/layouts/_header.html.slim [hunk 1]
```diff
@@ -1,15 +1,15 @@
-nav.navbar
+nav.navbar.is-dark role="navigation" aria-labe="main navigation"
```
Comment (rueyaa332266) [inline app/views/layouts/_header.html.slim:1]: Please fix the typo.

**Comment by rueyaa332266** [comment]
Would you also add the logo in README?

File: app/views/layouts/_header.html.slim (hunk 1 above)
Comment (naotospace) [reply app/views/layouts/_header.html.slim]: Fixed!

**Comment by naotospace** [comment]
Thanks for review.
I fixed and added logo also.

**Review by rueyaa332266 — APPROVED** [review]
LGTM

**Pull request merged**